wandb/sdk/launch/builder/kaniko_builder.py

```diff
@@ -148,13 +151,15 @@ def from_config(
             raise LaunchError(
                 "Builder config must include 'type':'kaniko' to create a KanikoBuilder."
             )
-        build_context_store = config.get("build-context-store")
+        build_context_store = config.get("build-context-store", "")
         if build_context_store is None:
-            raise LaunchError(
-                "You are required to specify an external build "
-                "context store for Kaniko builds. Please specify a  "
-                "storage url in the 'build_context_store' field of your builder config."
-            )
+            if not PVC_MOUNT_PATH:
+                raise LaunchError(
+                    "You must specify a build context store for kaniko builds. "
+                    "You can set builder.build-context-store in your agent config "
+                    "to a valid s3, gcs, or azure blog storage URI. Or, configure "
+                    "a persistent volume claim through the agent helm chart."
```

Comment:
nit: should we link to the helm chart here?

Reply:
yes adding that